context/app/static/js/components/workspaces/WorkspacesList.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useContext, useState, useEffect } from 'react';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import styled from 'styled-components';
+
+import { AppContext } from 'js/components/Providers';
+import { DeleteIcon, AddIcon } from 'js/shared-styles/icons';
+import { WhiteBackgroundIconButton } from 'js/shared-styles/buttons';
+
+import { SeparatedFlexRow, FlexBottom } from './style';
+import { createNotebookWorkspace } from './utils';
+
+// TODO: Copy-and-paste from SummaryData/style
+const StyledButton = styled(WhiteBackgroundIconButton)`
+  height: 36px;
+`;
+
+function useWorkspacesList() {
```

Comment:
We've used separate files for hooks and styles and I'd prefer to stay consistent until we've decided to move away from that.